server/test/unit/server/webservices/test_urls.py

```diff
@@ -21,4 +24,26 @@ def test_match_type_resource_view(self):
         pass
 
     def test_match_types_view(self):
-        pass
+        """
+        Test url matching for /plugins/types/.
+        """
+        match = resolve('/v2/plugins/types/')
+
+        self.assertEqual(match.url_name, 'plugin_types')
+
+        should_not_match = [
```

Comment:
Based on yesterday's convo I don't think we should do negative matching but we should ensure:

expected view name + specific args --> produces correct URL
AND
that correct url when resolved will produce the correct view name.